.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+# ******** NOTE ********
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, brand-consolidation, production ]
```

Comment:
might want to clean up these branch names since they don't exist

Reply:
I meant to follow up on this as well. Is there a branch other than master we should be regularly scanning? If not I'll delete the other two and use master.

Reply:
i think `master` is the only long lived branch, the rest are feature branches. idk if we want to scan those during dev work but probably master only is a good starting place